series/supportedseries_test.go

```diff
@@ -22,6 +24,13 @@ func (s *supportedSeriesSuite) SetUpTest(c *gc.C) {
 	s.CleanupSuite.SetUpTest(c)
 	cleanup := series.SetSeriesVersions(make(map[string]string))
 	s.AddCleanup(func(*gc.C) { cleanup() })
+
+	series.RestoreUbuntuSeries()
+	s.AddCleanup(func(*gc.C) { series.RestoreUbuntuSeries() })
+
+	s.PatchValue(series.TimeNow, func() time.Time {
+		return time.Date(2020, 7, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Where do we test for this patched value?

Reply:
We aren't comparing for it in the test itself, it's just that the hard-coded distro info has an EOL date in it, and the code is comparing against that. Specifically the EOL for eoan had elapsed. And I didn't want the unit tests to keep breaking as things go EOL -- we're still testing the right code paths.